### PR TITLE
Flake8 for xenial

### PR DESCRIPTION
**Pull request by SteveMacenski**

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -1038,6 +1038,7 @@ python-flake8:
   ubuntu:
     trusty: [python-flake8]
     trusty_python3: [python3-flake8]
+    xenial: [python-flake8]
```
Comment (mikaelarguedas) [inline rosdep/python.yaml]: as this package is available on [all active ubuntu distributions](https://packages.ubuntu.com/trusty/python-flake8), can you please update this to cover them all?
```yaml
  ubuntu:
    '*': [python-flake8]
    trusty_python3: [python3-flake8]
```
should do the trick

**Comment by SteveMacenski** [comment]
done

**Review by mikaelarguedas — APPROVED** [review]
lgtm, thanks @SteveMacenski

File: rosdep/python.yaml (hunk 1 above)
Comment (mikaelarguedas) [reply rosdep/python.yaml]: Python3 rules are added upon request from users using them. As it;s not been requested for 4 years I'm not planning on adding them here

**Review by tfoote — APPROVED** [review]

**Pull request merged**